bot.js

```diff
@@ -119,44 +119,43 @@ client.on("chat", function(channel, user, message, self) {
   }
 });
 
-// SUB REACTION MODULE
-client.on("subscription", function(channel, username) {
+// SUB REACTION MODULE (moved raffle things)
+client.on("subscription", function(channel, username, methods, message, userstate) {
   client.say(channel, "New sub PogChamp");
+  //raffle thing in #smaczny channel
+  if (channel !== "#smaczny") return;
+  if (username === client.username) return; //escapes own subs
+  sendMsg("smaczny", "!raffle 3500");
 });
 
 client.on(
   "subgift",
   (channel, username, streakMonths, recipient, methods, userstate) => {
     client.say(channel, "Gifters PogChamp ");
-  }
-);
+    //raffle thing in #smaczny channel
+    if (channel !== "#smaczny") return;
+    if (recipient === client.username) return; //escapes own subs
+    sendMsg("smaczny", "!raffle 3500");
+});
 
 client.on(
   "submysterygift",
   (channel, username, numbOfSubs, methods, userstate) => {
-    client.say(channel, "GIFTERS PogChamp ");
-  }
-);
+    client.say(channel, "ANON GIFTERS PogChamp ");
```

Comment:
submysterygift isnt necessarily anon, it's "Krakenbul is gifting 100 Tier 1 subscriptions to this channel." and then 100 subgift messages roll in.

Reply:
Oh yes, I was refering to the fact, that you can't get the information about who got that sub, but still, you're right about that one.